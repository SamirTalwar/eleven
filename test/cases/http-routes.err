Request: {"method":"GET","path":"/"}
Response: {"status":200,"body":"Hello, world!"}
Request: {"method":"GET","path":"/health"}
Response: {"status":200,"body":"Everything is awesome."}
Request: {"method":"GET","path":"/not-found"}
Response: {"status":404,"body":""}
Stopping...
Stopped.
