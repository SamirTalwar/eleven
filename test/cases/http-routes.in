http --check-status --print='hb' GET :8080 | dos2unix | grep -Ev '^Date: '
echo
http --check-status --print='hb' GET :8080/health | dos2unix | grep -Ev '^Date: '
echo
http --print='hb' GET :8080/not-found | dos2unix | grep -Ev '^Date: '
echo
