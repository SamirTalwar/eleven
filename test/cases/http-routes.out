HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Hello, world!

HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Everything is awesome.

HTTP/1.1 404 Not Found
Connection: keep-alive
Transfer-Encoding: chunked


